src/features/dailyNotesPopup.tsx

```diff
@@ -97,21 +96,23 @@ const DailyNotesPopup = ({ onClose }: RoamOverlayProps<{}>) => {
             parentUid: pageUid,
           })
       ).then((uid) => {
-        const { windowId } = getUids(
-          containerRef.current.querySelector(".roam-block")
-        );
-        window.roamAlphaAPI.ui.setBlockFocusAndSelection({
-          location: {
-            "block-uid": uid,
-            "window-id": windowId,
-          },
-        });
+        const blockElement = containerRef.current?.querySelector(
+          ".roam-block"
+        ) as HTMLDivElement;
+        if (blockElement) {
+          const { windowId } = getUids(blockElement);
+          window.roamAlphaAPI.ui.setBlockFocusAndSelection({
+            location: {
+              "block-uid": uid,
+              "window-id": windowId,
+            },
+          });
+        }
       });
     } else if (!loaded) {
       setLoaded(true);
     } else if (minimizedRef.current) {
       setLoaded(false);
-      containerRef.current = undefined;
```

Comment:
Was this required?  Typescript said it wasn't possible to set current, because it's read only.
Removing it didn't seem to affect the dnp popup function.